frontend/test/metabase/scenarios/question/reproductions/17514-ui-overlay.js

```diff
@@ -142,6 +142,7 @@ export function issue17514() {
 
         visualize();
         cy.findByTextEnsureVisible("Subtotal");
+        cy.wait("@dataset");
```

Comment:
@iethree please revert this change.
`visualize()` helper already waits for `@dataset` so this will make things only worse.

Looking at the screenshot for the failed test, it doesn't seem like a flake but a legitimate failure because object detail window now appears.
![image](https://user-images.githubusercontent.com/31325167/165545258-bf839b55-2388-4168-bd6e-a5932ce251c9.png)

In order to investigate failures in the CI, please go to the failed run and then [click on Artifacts tab](https://app.circleci.com/pipelines/github/metabase/metabase/34791/workflows/d81f915f-30b0-470f-807d-27a77be2a4ef/jobs/1807973/artifacts). You'll see both screenshots and videos there.

![image](https://user-images.githubusercontent.com/31325167/165545485-ac9d854e-73d9-4fd8-8037-81eaccf79b2d.png)

Reply:
The failure is because it's trying to click on `110.93`, but it actually clicks on the id column because it clicks before the UI is fully loaded and there's a small visual shift.  Any suggestions on how to wait for that UI load more accurately?

Reply:
Well, that's a bummer then! :(
Up until fairly recently it was enough to wait for the `response` (in this case `POST /api/dataset`), but then we started having more and more these visual glitches and it takes browser a bit more to actually render results.

To make things worse, this test already implements a "second level" guard by additionally making sure that the table header is rendered:
`cy.findByTextEnsureVisible("Subtotal");`

I don't have any other ideas other than try to make sure 110.93 is visible before clicking on it.
`cy.findByTextEnsureVisible("110.93");`

That's still not going to be a guarantee, but it's the best we can do at the moment.

Reply:
@nemanjaglumac take a look at the latest commit. What we're testing here is whether the table UI is clickable at all - so instead of having the assertion on a table cell, I've updated the assertion to be on the table header which seems less prone to shifting.

Reply:
Works for me.
Great thinking. Thanks